python/pip-wheel-metadata/pyarrow.dist-info/LICENSE.txt

```diff
@@ -0,0 +1,87 @@
+
```

Comment:
sorry in my previous comment I was asking generally if the pip-wheel stuff is required in this PR or is the python code enough (I'm don't have any experience with wheel packaging).

Reply:
If they are required you have to add an apache header or exclude them, so CI passes.